tests/org.komodo.relational.test/src/org/komodo/relational/model/internal/ParameterImplTest.java

```diff
@@ -17,6 +17,7 @@
 import org.junit.Test;
 import org.komodo.relational.RelationalConstants;
 import org.komodo.relational.RelationalConstants.Nullable;
+import org.komodo.relational.RelationalObject.Filter;
```

Comment:
Something like this should not really be committed. It just adds noise to the commit. Better to revert these classes and only leave those where a change has actually been made.